CMakeLists.txt

```diff
@@ -49,8 +49,8 @@ message(STATUS "Compilation date = XX${DATE_RESULT}XX")
 #
 set(FREEDV_VERSION_MAJOR 1)
 set(FREEDV_VERSION_MINOR 6)
-set(FREEDV_VERSION_PATCH 1)
-set(FREEDV_VERSION_SUFFIX "")
+set(FREEDV_VERSION_PATCH 2)
+set(FREEDV_VERSION_SUFFIX "pulseaudio-devel")
```

Comment:
If we are going to merge should this be a numeric patch level?  Or even bump minor?

Reply:
It's enough of a change in the backend code that bumping minor seems reasonable here. We'd probably also want to spin a new release immediately after merge to get some runtime.

Reply:
Bumped up to 1.7.0. 👍